docs/development/getting-started.rst

```diff
@@ -0,0 +1,87 @@
+***********
+Development
+***********
+
+To set up the your development enviroment you will need to first install project deps:
+
+Linux:
+######
+.. code-block:: bash
+
+   Run the env install script: sudo sh scripts/libgit2.sh
+   pip install -r requirements.txt
+   pip install -r test-requirements.txt
+
+OS X:
+#####
+.. code-block:: bash
+
+   brew install libgit2
+   pip install pygit2==0.25.0
+   pip install -r requirements.txt
+   pip install -r test-requirements.txt
+
+
+
+.. note:: this will install latest libgit2 library so you have to make sure you install the same version library with pip ( current version: 0.25.0 )
+
+
+Virtualenv
```

Comment:
I would strongly suggest _just_ to use virtualenv, and install the others directly only if really needed. Secondly, I would suggest that the main use to be from the Docker container. If lots of options are provided, then lots of options will need to be supported. Limiting the scope early would be my recommendation.